CheckValues/Results.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Text;
+using WaterLog_Backend;
+using WebApplication1;
+using WaterLog_Backend.Models;
+using WaterLog_Backend.Controllers;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CheckValues
+{
+    public class Results
+    {
+        public MonitorsController getController()
+        {
+            string connection = "";
+            IConfiguration chosenconfig;
+            var jsonconfig = new ConfigurationBuilder()
+            .AddJsonFile("appsettings.json")
```

Comment:
I don't think this is correct, usually there's a tab when calling more functions on an object.